source/clear-linux/reference/image-types.rst

```diff
@@ -35,14 +35,11 @@ Table 2 lists the currently available images that are platform specific.
    * - Image Type
      - Description
 
-   * - installer.img
-     - Preferred image of |CL| with interactive installer.
+   * - live-desktop.img or live-desktop.iso
+     - Image for live booting desktop version into memory, without requiring installaton.
```

Comment:
The live-desktop will boot into a gnome desktop environment.  The .iso image is read-only and nothing is saved from the session.  If you use the .img to create the USB (using dd or something like that, it creates a RW USB image and changes are saved.  @gtkramer , correct me if that is wrong.

Reply:
That is correct.  We produce the .img for the live-desktop and live-server for this RW capability, but we don't advertise because that is not what marketing wants